typescript/utils/src/math.test.ts

```diff
@@ -0,0 +1,48 @@
+import { expect } from 'chai';
+
+import { mean, median, randomInt, stdDev, sum } from '../src/math.js';
```

Comment:
Sorry I missed this in my first pass. This import path should be just `'./math.js'`

Reply:
done